Makefile

```diff
@@ -157,4 +157,4 @@ integration_test_legacy: integration_build
 	MOZ_HEADLESS=1 $(COMPOSE_INTEGRATION) run firefox sh -c "sudo chmod a+rwx /code/app/tests/integration/.tox;tox -c app/tests/integration -e integration-test-legacy $(TOX_ARGS) -- -n 4 $(PYTEST_ARGS)"
 
 integration_test_nimbus: integration_build
-	MOZ_HEADLESS=1 $(COMPOSE_INTEGRATION) run firefox sh -c "sudo chmod a+rwx /code/app/tests/integration/.tox;tox -c app/tests/integration -e integration-test-nimbus $(TOX_ARGS) -- -n 4 $(PYTEST_ARGS)"
+	$(COMPOSE_INTEGRATION) up -d; MOZ_HEADLESS=1 $(COMPOSE_INTEGRATION) run firefox sh -c "sudo chmod -R a+rwx /code/app/tests/integration/;tox -c app/tests/integration -e integration-test-nimbus $(TOX_ARGS) -- --html=report.html --self-contained-html  $(PYTEST_ARGS)"
```

Comment:
Oh I don't think you should add this here?  I kept starting the environment and running the tests separate on purpose.  Does this have to be here?

Reply:
No, it shouldn't my bad!